spec/views/batch/edit.html.erb_spec.rb

```diff
@@ -1,8 +1,12 @@
 require 'spec_helper'
 
 describe 'batch/edit.html.erb' do
-  let(:batch) { stub_model(Batch, id: '123') }
-  let(:generic_file) { stub_model(GenericFile, id: nil, depositor: 'bob', rights: ['']) }
+  let(:batch) { Batch.create }
+  let(:generic_file) do
+    GenericFile.new(title: ['some title']).tap do |f|
```

Comment:
@mjgiarlo it doesn't need to be save. We just need to have an actual batch in hand for the test.